packages/change-case/package.json

```diff
@@ -45,6 +45,9 @@
     "specs": "ts-scripts specs",
     "test": "ts-scripts test"
   },
+  "dependencies": {
+    "is-plain-obj": "^4.1.0"
```

Comment:
I would inline it to not introduce extra dependencies.

Btw hi dude!

Reply:
Updated 🙂 

Hope you're doing well @TrySound !